### DISCUSSION TITLE
skoala/registry/managed/governance/flow-control/

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#1570] [Announcements]
# skoala/registry/managed/governance/flow-control/

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/skoala/registry/managed/governance/flow-control/

**Comment by liyuerich** [comment]
能每个规则都给个样例吗？

**Comment by liyuerich** [comment]
来源应用：Sentinel 可以针对调用者进行限流，默认为 default，表示不区分来源，适用于所有的调用者。
非默认的能给个例子吗？

**Comment by liyuerich** [comment]
QPS和线程数能分别介绍吗？ 因为QPS的高级配置和线程数不一样
